rskj-core/src/main/java/org/ethereum/rpc/parameters/BlockIdentifierParam.java

```diff
@@ -0,0 +1,50 @@
+package org.ethereum.rpc.parameters;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.ethereum.rpc.exception.RskJsonRpcRequestException;
+import org.ethereum.util.Utils;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+
+@JsonDeserialize(using = BlockIdentifierParam.Deserializer.class)
+public class BlockIdentifierParam implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private static final List<String> STRING_IDENTIFIERS = Arrays.asList("earliest", "latest", "pending");
```

Comment:
These definitions are being repeated in other places. I thin we should have a common class for them. This is a proposal I did in other PR to solve a similar issue. https://github.com/rsksmart/rskj/pull/2080/files#diff-e15384bdcacb6015dc99ec6de04a4cf59bcab14c86685b1174a995653e88d41b